### ISSUE TITLE
Gets super hot and kernel panic happens leading to reboot

### ISSUE THREAD

**Issue by testdil** [checkra1n/BugTracker#768]
**Tell us about your setup:**
1. What iDevice are you using?
iPhone 7 Plus 
2. On what version of iOS is it?
13.3
3. What version of checkra1n are you using? GUI or CLI?
9.7 beta
4. What is your host system (OS version? Hackintosh? VM? etc.)?
Mojave 2012 MacBook Pro 
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
Lightning to USB

**What are the steps to reproduce the issue?**
1. Use phone for 10-20days
2. Wait for it too heat up
3. ???
4. Resets 
...

**What do you expect, and what is happening instead?**
I expect it to not kernel panic

**Does the issue also occur if you enable Safe Mode?**
Cant test 

**Any other info, error logs, screenshots, ...?**
https://pastebin.com/RG6YYFeJ

**Comment by Siguza** [comment]
This sounds a lot like #533, given the amount of suspended processes in your panic log.  
Do you have the latest Substrate update installed?

**Comment by testdil** [comment]
Yes I’m on 0.9.7100 should I reinstall it?

**Comment by Siguza** [comment]
I mean it shouldn't hurt, but I don't feel like it's gonna help either if you're already on that version. :/  
I'm afraid I don't have a good idea on how to even debug this issue. All I can tell from the log is that the reason for the panic is definitely substrate not running anymore. But I don't even know whether it was a rogue process that killed it, or whether it crashed due to a fault of its own...

Marking this as deferred for the time being.

**Comment by testdil** [comment]
I have 2 more panic logs if those could help?

**Comment by Siguza** [comment]
I guess they won't hurt...

**Comment by testdil** [comment]
https://pastebin.com/dQGZnKpL

https://pastebin.com/tZRAEq4E

**Comment by Siguza** [comment]
Yeah, these look pretty much the same. No substrate processes, but a ton of suspended processes... :/

**Comment by HaGii** [comment]
Did you manage to solve the problem? 
I got the same issue, even with the newest version of Cydia Substrate, which is 0.9.7101.

**Comment by testdil** [comment]
> Did you manage to solve the problem?
> I got the same issue, even with the newest version of Cydia Substrate, which is 0.9.7101.

I read some where on reddit that disabling tweak injection into the camera app might help. So far it seams to have worked but not 100% sure.

**Comment by HaGii** [comment]
Did you manage to solve the problem? 
I got the same issue, even with the newest version of Cydia Substrate.

**Comment by HaGii** [comment]
> > Did you manage to solve the problem?
> > I got the same issue, even with the newest version of Cydia Substrate, which is 0.9.7101.
> 
> I read some where on reddit that disabling tweak injection into the camera app might help. So far it seams to have worked but not 100% sure.

Can you please explain how you did that?
